### ISSUE TITLE
Feature enhancement: `ctrl + click` option to follow URLs in editor

### ISSUE THREAD

**Issue by marcuslehr** [notepad-plus-plus/notepad-plus-plus#12246]
### Description of the Issue
It's nice that notepad++ has a clickable link feature, however, attempting to edit the links is cumbersome since you cannot put your cursor within it using the mouse. 

### Steps to Reproduce the Issue
1. Type a URL into the editor
2. Attempt to place cursor within URL using the mouse

### Expected Behavior
Some way to place the cursor into the URL string

### Actual Behavior
There is no way. Clicking outside the URL and using the arrow keys is a work around. Or disable clickable links entirely.

### Proposed Enhancement
Add an option- preferably as the default- to use `ctrl + click` to open the URL in the browser. If control is not held while clicking, the cursor will be placed where the mouse is like normal. This behavior is common in office editors and it would be natural to follow suit here.

**Comment by ArkadiuszMichalski** [comment]
Duplicate of https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9993.

**Comment by alankilborn** [comment]
>attempting to edit the links is cumbersome since you cannot put your cursor within it using the mouse

Why can't you?
Point to the place in the link text that you want to start editing, then single left-click with no modifier keys -- this is just like moving the caret with the mouse in any text.
Caret ends up right in the link text where you clicked.
Tried in v8.4.6 with default settings.
Since OP ignored the issue template's request for _Debug Info_, we don't know what version he is using.

**Comment by ArkadiuszMichalski** [comment]
Right, maybe there is a problem with mouse (double click instead of one)? Please also update debug info.

**Comment by alankilborn** [comment]
@ArkadiuszMichalski I wasn't advocating for reopening; as there is no problem with using the mouse to click into underlined text to edit it (the problem statement of this issue).

**Comment by ArkadiuszMichalski** [comment]
>there is no problem with using the mouse to click into underlined text to edit it (the problem statement of this issue).

If someone has bothered here and opened an issue, something does not work for him (maybe plugins, maybe configuration, or something else). If there is no respone, I will close it.

**Comment by aczire** [comment]
Well, I encountered a similar problem. You can double click to select a word, but double click to select a word inside link is not possible. 
How can I force to open link only with a modifier key (like only open link if ctrl is pressed)?
or
How can I disable double click to open links?

**Comment by alankilborn** [comment]
@aczire 

---

>How can I force to open link only with a modifier key (like only open link if ctrl is pressed)?

Not possible (as this issue implies)... well, not possible without going to the trouble of scripting something.  

---

>How can I disable double click to open links?

Uncheckmark this:

![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/77065706/16e1508c-bd7f-4ad0-8f5e-17a21eaa27ef)

**Comment by aczire** [comment]
@alankilborn Thank you!

What i was hoping for is disable single click/double click on links. Not disable links altogether. 

I would recommend a "ctrl+click to open links" in Preferences. Then it will not affect any text editing features in the application (for example double click to select a word inside the link), since that is the primary objective of the text editor.

**Comment by htp-earthwormjim** [comment]
bump from the crypt
